packages/ui/src/components/pipeline-nodes/splitview-stage-node.tsx

```diff
@@ -56,7 +56,7 @@ export function SplitView_StageNode(props: SplitView_StageNodeProps) {
       <ExecutionStatus executionStatus={executionStatus} />
 
       <div
-        className={cn('bg-cn-background-8 rounded-md', {
+        className={cn('bg-cn-background-softgray rounded-md', {
```

Comment:
this rename doesn't make sense.. this color will not be "soft gray" in other themes.. that's why we were using numbers. 
we should use semantic colors instead if possible.
cc @praneshg239